rasa/core/tracker_store.py

```diff
@@ -282,9 +281,9 @@ def retrieve(self, sender_id: Text) -> Optional[DialogueStateTracker]:
         if sender_id in self.store:
             logger.debug(f"Recreating tracker for id '{sender_id}'")
             return self.deserialise_tracker(sender_id, self.store[sender_id])
-        else:
-            logger.debug(f"Creating a new tracker for id '{sender_id}'.")
-            return None
+
+        logger.debug(f"Creating a new tracker for id '{sender_id}'.")
```

Comment:
This comment doesn't make sense for me at this location. We should either say that there was no tracker for the user or move the log to where the tracker is actually created.